vizro-core/tests/unit/vizro/models/test_page.py

```diff
@@ -101,7 +101,7 @@ def test_valid_component_types(self, standard_px_chart):
         [vm.Checklist(), vm.Dropdown(), vm.RadioItems(), vm.RangeSlider(), vm.Slider()],
     )
     def test_invalid_component_types(self, test_component):
-        with pytest.raises(ValidationError, match=re.escape("(allowed values: 'button', 'card', 'graph')")):
+        with pytest.raises(ValidationError, match=re.escape("allowed values: ")):
```

Comment:
Do we not want to test for the actual components?

Reply:
I had to change the test just because I added a component so found that too specific. Ideally it checks for the type of error message, but does not need to be adjusted just because you added a new component

Reply:
Tbh I think this test is not going to stop anyone adding another component in the allowed types. If the reviewer and contributor know about this restriction, they will forbid it even without this test, if not, they will change the test.

I added it back in, but I think it's not really our place to test automatically generated error messages as a reminder. If we really care about this, why not add a note behind the component type, which is much clearer and less annoying?

Reply:
But this would require that the reviewer and contributor remember this restriction. 

It just requires adding the str type in there, which I think is okay? We can also leave this thread open - I am not strictly against it if the team prefers that. The comment next to the `ControlType` and `ComponentType` would also be okay for me, though I still prefer the tests as I think they serve as documentation as well. And there I prefer to be more explicit in showing what's allowed and not allowed.